src/saml/User.php

```diff
@@ -22,19 +22,16 @@ public static function convertToSamlFieldNames(
             'eduPersonPrincipalName' => [
                 $username . '@' . $idpDomainName,
             ],
-            
+
             /**
-             * Misspelled version of eduPersonTargetedID.
+             * Misspelled version of eduPersonTargetedID. (Accidentally used in the past)
              * @deprecated
              */
             'eduPersonTargetID' => (array)$uuid, // Incorrect, deprecated
-            
-            /**
-             * NOTE: Do NOT include eduPersonTargetedID. If you need it, use the
-             * core:TargetedID module (at the Hub, if using one) to generate an
-             * eduPersonTargetedID.
-             */
-            
+
+            // Proper spelling of eduPersonTargetedID.
+            'eduPersonTargetedID' => (array)$uuid,
```

Comment:
No, this is contrary to the design for that attribute field.

Reply:
If we want to include the eduPersonTargetedID (which seems like it could very well be useful), lets use the ["core:TargetedID" authproc](https://simplesamlphp.org/docs/latest/core/authproc_targetedid.html) from SimpleSAMLphp.

We'll just need to figure out where to put that authproc (maybe we can put it on the hub and/or the IDPs) and what attribute it should use to generate the eduPersonTargetedID.